execution_engine_testing/tests/src/test/system_contracts/auction/bids.rs

```diff
@@ -3068,3 +3082,999 @@ fn should_fail_with_more_accounts_than_slots() {
 fn should_run_genesis_with_exact_validator_slots() {
     check_validator_slots_for_accounts(DEFAULT_EXEC_CONFIG.validator_slots() as usize);
 }
+
+#[ignore]
+#[test]
+fn should_delegate_and_redelegate() {
+    let system_fund_request = ExecuteRequestBuilder::standard(
+        *DEFAULT_ACCOUNT_ADDR,
+        CONTRACT_TRANSFER_TO_ACCOUNT,
+        runtime_args! {
+            ARG_TARGET => *SYSTEM_ADDR,
+            ARG_AMOUNT => U512::from(SYSTEM_TRANSFER_AMOUNT)
+        },
+    )
+    .build();
```

Comment:
We're writing this down so many times that I am surprised there's no helper method like
```
fund(from_account, to_account, amount)
```

Reply:
If not in this PR, let's create a ticket to add most commonly-used helpers like `fund`, `bid` `(un)delegate`, `withdraw`, etc. and use them across all tests. If we keep postponing that, people will never change their habits.